lib/spack/spack/build_systems/autotools.py

```diff
@@ -151,13 +151,25 @@ def configure(self, spec, prefix):
         options = ['--prefix={0}'.format(prefix)] + self.configure_args()
         inspect.getmodule(self).configure(*options)
 
+    def build_targets(self):
```

Comment:
A suggestion for a final polishing: could you try to make this a normal attribute?
```python
build_targets = []
```